values_community_edition.yaml

```diff
@@ -29,6 +29,7 @@ gateway:
   hostName: "gateway.tykbeta.com"
   tls: true
   containerPort: 8080
+  controlApiPort: 9696
```

Comment:
By default I think this is the same port that the gateway is listening on, so we should set these in such a way that it only overrides if someone intentionally changes it.

Reply:
9696 is the default port for the gateway api, moved this to the `values.yaml` file to have the default port explicit (if that's what you meant)